common/src/main/java/org/opensearch/ml/common/model/Guardrails.java

```diff
@@ -22,26 +22,22 @@
 @Getter
 public class Guardrails implements ToXContentObject {
     public static final String TYPE_FIELD = "type";
-    public static final String ENGLISH_DETECTION_ENABLED_FIELD = "english_detection_enabled";
```

Comment:
Just curious to know why are we removing this?

Reply:
Yeah, I got comments before to remove it as we don't implement this feature right now. So to not confuse users, remove it now.